pkg/controller/util.go

```diff
@@ -250,6 +225,8 @@ func MakeImporterPodSpec(image, verbose, pullPolicy, ep, secret string, pvc *v1.
 			},
 			Labels: map[string]string{
 				CDI_LABEL_KEY: CDI_LABEL_VALUE,
+				// this label is used when searching for a pvc's import pod.
+				AnnImportPVC: pvc.Name,
```

Comment:
namespace is not needed?

Reply:
right we don't have to specify the namespace here. The namespace is set during the create action

example
```
pod, err := client.CoreV1().Pods(namespace).Create(pod)
```